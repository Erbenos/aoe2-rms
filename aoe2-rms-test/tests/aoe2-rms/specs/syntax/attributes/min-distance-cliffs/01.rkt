#lang aoe2-rms
; Create cliffs that are more curly than usual.

<CLIFF-GENERATION>
(min-number-of-cliffs 9999)
(max-number-of-cliffs 9999)
(min-distance-cliffs 1)
