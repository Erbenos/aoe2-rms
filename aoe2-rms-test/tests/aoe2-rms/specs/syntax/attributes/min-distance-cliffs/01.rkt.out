<CLIFF_GENERATION>
min_number_of_cliffs 9999
max_number_of_cliffs 9999
min_distance_cliffs 1
